main/autogen/autogen.cc

```diff
@@ -674,4 +687,34 @@ void ParsedFile::classlist(core::Context ctx, vector<string> &out) {
     }
 }
 
+void ParsedFile::subclasses(core::Context ctx, map<string, set<string>> &out) {
```

Comment:
please use `UnorderedSet` and `UnorderedMap`

Reply:
@DarkDimius I want the final output to be sorted (I iterate over the `map` keys in order, and for each map key, I iterate over the corresponding `set` members in order). I wanted to avoid `fast_sort`ing the `set` members in that loop. The BigO seems equivalent to me in either case, but using the ordered data structures was more convenient. Let me know if I'm missing something.